test/unit/rules-elements.html

```diff
@@ -0,0 +1,192 @@
+<script src="https://cdnjs.cloudflare.com/ajax/libs/validator/4.0.5/validator.js"></script>
```

Comment:
Bad idea to include it from CDN, there should be possibility to test everything completely offline.
Also what is the need to include something from CDN at all?
Unit tests should be simple, short and self-contained.